llama_index/embeddings/base.py

```diff
@@ -52,6 +52,7 @@ def __init__(
         self,
         embed_batch_size: int = DEFAULT_EMBED_BATCH_SIZE,
         tokenizer: Optional[Callable] = None,
+        **kwargs
```

Comment:
i don't think this needs to be in the baseembedding class, only in the openai class

Reply:
If this is not present, the constructor could pass unknown params to base which triggers an error. Is there a better way to handle that?

Reply:
You can modify the openai embedding constructor to only pass the `embed_batch_size` and `tokenizer` to the super class.

Reply:
Was thinking that more params might be needed in Cohere and other embeddings as well. This would work across the board now, similar to how it works for LLMPredictors

Reply:
> Thanks @siddhant01. Makes for cleaner code, have made the change now

@roh26it tbh I still think it's cleaner to have just on openai class right now, esp since we don't support arbitrary kwargs for other embedding classes yet. We can add to base class later if it makes sense, right now it's just for openai